gradle/dependencies.gradle

```diff
@@ -39,11 +38,9 @@ ext.dep = [
         runtime: "org.antlr:antlr4-runtime:$versions.antlr4",
     ],
     apollo                : [
-        plugin          : "com.apollographql.apollo:apollo-gradle-plugin:$versions.apollo",
-        runtime         : "com.apollographql.apollo:apollo-runtime:$versions.apollo",
-        rx2             : "com.apollographql.apollo:apollo-rx2-support:$versions.apollo",
```

Comment:
these were never used since integration tests use the real module instead of latest maven dependency.